### DISCUSSION TITLE
Plots not starting - No errors

### DISCUSSION THREAD

**Discussion by ludgats** [swar/Swar-Chia-Plot-Manager#702] [Q&A]
New jobs not starting after updating to new version.

I have wiped it and started from fresh. I have no errors.

**Comment by Meyer85** [comment]
I'm sitting in the same boat

Viewer working perfectly, no jobs get started. If I start a job in the GUI it shows up in the viewer.

Please help I'm getting desperate



chia_location: C:\Users\Plotter\AppData\Local\chia-blockchain\app-1.1.6\resources\app.asar.unpacked\daemon\chia.exe


manager:
  check_interval: 60
  log_level: ERROR


log:
  folder_path: C:\Users\Plotter\.chia\mainnet\plotter


view:
  check_interval: 60
  datetime_format: "%Y-%m-%d %H:%M:%S"
  include_seconds_for_phase: false
  include_drive_info: true
  include_cpu: true
  include_ram: true
  include_plot_stats: true


notifications:
  # These are different settings in order to notified when the plot manager starts and when a plot has been completed.

  # DISCORD
  notify_discord: false
  discord_webhook_url: https://discord.com/api/webhooks/0000000000000000/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX

  # IFTTT, ref https://ifttt.com/maker_webhooks, and this function will send title as value1 and message as value2.
  notify_ifttt: false
  ifttt_webhook_url: https://maker.ifttt.com/trigger/{event}/with/key/{api_key}

  # PLAY AUDIO SOUND
  notify_sound: false
  song: audio.mp3

  # PUSHOVER PUSH SERVICE
  notify_pushover: false
  pushover_user_key: xx
  pushover_api_key: xx
  
  # TELEGRAM
  notify_telegram: false
  telegram_token: xxxxx

  # TWILIO
  notify_twilio: false
  twilio_account_sid: xxxxx
  twilio_auth_token: xxxxx
  twilio_from_phone: +1234657890
  twilio_to_phone: +1234657890


instrumentation:
  # This setting is here in case you wanted to enable instrumentation using Prometheus.
  prometheus_enabled: false
  prometheus_port: 9090


progress:
  phase1_line_end: 801
  phase2_line_end: 834
  phase3_line_end: 2474
  phase4_line_end: 2620
  phase1_weight: 33.4
  phase2_weight: 20.43
  phase3_weight: 42.29
  phase4_weight: 3.88


global:
  max_concurrent: 10
  max_for_phase_1: 4
  minimum_minutes_between_jobs: 5


jobs:
  - name: plotter
    max_plots: 999
    farmer_public_key:
    pool_public_key:
    temporary_directory: D:\Temp Plots
    temporary2_directory:
    destination_directory:
      - E:\Plots
      - F:\Plots
      - G:\Plots
      - H:\Plots
      - I:\Plots
      - J:\Plots
    size: 32
    bitfield: true
    threads: 2
    buckets: 128
    memory_buffer: 8000
    max_concurrent: 8
    max_concurrent_with_start_early: 9
    initial_delay_minutes: 1
    stagger_minutes: 14
    max_for_phase_1: 16
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 1
    temporary2_destination_sync: false
    exclude_final_directory: false
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 32
    enable_cpu_affinity: false
    cpu_affinity: [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60 ]

**Comment by VLIN08** [comment]
> New jobs not starting after updating to new version.
> 
> 
> 
> I have wiped it and started from fresh. I have no errors.

I am sitting in the same boat too even though I did not update. 

Did you figure out how to fix this? Thanks

**Reply by Meyer85** [reply]
skip_full_destinations: true

change to false

fixed it for me

**Reply by Eugenok** [reply]
This. Fixed for me too!

**Comment by Meyer85** [comment]
> New jobs not starting after updating to new version.
> 
> I have wiped it and started from fresh. I have no errors.

Check your config file

skip_full_destinations: true

change to false

fixed it for me

**Reply by ludgats** [reply]
This did not work for me unfortunately

**Comment by umair4848** [comment]
I have the same issue

**Comment by Kali123411** [comment]
I am having the same problem.

**Comment by ManuelRueedi** [comment]
Did some one fix it? 

Changing skip_full_destinations to false didn't work for me.
